src/modules/fw_att_control/FixedwingAttitudeControl.cpp

```diff
@@ -258,6 +258,10 @@ FixedwingAttitudeControl::vehicle_manual_poll()
 			if (_vcontrol_mode.flag_control_rattitude_enabled) {
 				if (fabsf(_manual.y) > _parameters.rattitude_thres || fabsf(_manual.x) > _parameters.rattitude_thres) {
 					_vcontrol_mode.flag_control_attitude_enabled = false;
+					_roll_ctrl.set_max_rate(_parameters.acro_max_x_rate_rad);
```

Comment:
You'll need to do this in the main loop for ACRO. Since rattitude actually just disables the control mode flag_control_attitude_enabled you could combine this with the other reset in a single check for `_flag_control_attitude_enabled_last != _vcontrol_mode.flag_control_attitude_enabled`, and then set the corresponding limits.

Reply:
good idea. done.